samples/estimation/df-chemistry/chemistry.py

```diff
@@ -513,10 +516,50 @@ def ndarray2d_to_string(arr):
 with open("resource_estimate.json", "w") as f:
     f.write(res.json)
 
+data = {
+    'Run name': [],
+    'T factory fraction': [],
+    'Runtime': [],
+    'Physical qubits': [],
+    'rQOPS': []
+}
+
+result_obj = json.loads(res.json)
+
+if isinstance(result_obj,dict):
+    result_obj = [ result_obj ]
+
+for item in result_obj:
+    # Run name
+    data['Run name'].append(item["jobParams"]["qubitParams"]["name"])
+    # T factory fraction and Runtime
+    if "physicalCountsFormatted" in item:
+        data['T factory fraction'].append(item["physicalCountsFormatted"]["physicalQubitsForTfactoriesPercentage"])
+        data['Runtime'].append(item["physicalCountsFormatted"]["runtime"])
+    elif "frontierEntries" in item:
+        data['T factory fraction'].append(item["frontierEntries"][0]["physicalCountsFormatted"]["physicalQubitsForTfactoriesPercentage"])
+        data['Runtime'].item.append(item["frontierEntries"][0]["physicalCountsFormatted"]["runtime"])
+    else:
+        data['T factory fraction'].append("-")
+        data['Runtime'].append("-")
+    # Physical qubits and rQOPS
+    if "physicalCounts" in item:
+        data['Physical qubits'].append(item["physicalCounts"]["physicalQubits"])
+        data['rQOPS'].append(item["physicalCounts"]["rqops"])
+    elif "frontierEntries" in item:
+        data['Physical qubits'].append(item["frontierEntries"][0]["physicalCounts"]["physicalQubits"])
+        data['rQOPS'].append(item["frontierEntries"][0]["physicalCounts"]["rqops"])
+    else:
+        data['Physical qubits'].append("-")
+        data['rQOPS'].append("-")
+ 
+df = pd.DataFrame(data)
```

Comment:
I suggest to explicitly list the column names in the `columns` keyword parameter to `DataFrame`, because the order in the `dict` may not be deterministic.